Sming/Libraries/ModbusMaster/ModbusMaster.patch

```diff
@@ -1,13 +1,26 @@
+diff --git a/src/ModbusMaster.cpp b/src/ModbusMaster.cpp
+index cdcc534..f6f93d4 100644
+--- a/src/ModbusMaster.cpp
++++ b/src/ModbusMaster.cpp
+@@ -860,7 +860,7 @@ uint8_t ModbusMaster::ModbusMasterTransaction(uint8_t u8MBFunction)
+           break;
+       }
+     }
+-    if ((millis() - u32StartTime) > ku16MBResponseTimeout)
++    if ((millis() - u32StartTime) > MB_RESPONSE_TIMEOUT)
+     {
+       u8MBStatus = ku8MBResponseTimedOut;
+     }
 diff --git a/src/ModbusMaster.h b/src/ModbusMaster.h
-index 8ab0d52..f3b1a50 100644
+index 8ab0d52..c335790 100644
 --- a/src/ModbusMaster.h
 +++ b/src/ModbusMaster.h
 @@ -255,7 +255,7 @@ class ModbusMaster
      static const uint8_t ku8MBReadWriteMultipleRegisters = 0x17; ///< Modbus function 0x17 Read Write Multiple Registers
      
      // Modbus timeout [milliseconds]
 -    static const uint16_t ku16MBResponseTimeout          = 2000; ///< Modbus timeout [milliseconds]
-+    static const uint16_t ku16MBResponseTimeout          = MB_RESPONSE_TIMEOUT; ///< Modbus timeout [milliseconds]
++    //static const uint16_t ku16MBResponseTimeout          = 2000; ///< Modbus timeout [milliseconds]
      
```

Comment:
Perhaps just delete these lines, rather than comment them out?

Reply:
I kept them to preserve consistency with the original repo, since it is not my work.

Reply:
> with the original repo, since it is not my work.

Here we are talking about the patch file so it is fine to remove the commented line. @kmihaylov ping me when you have committed this change so that I can review and merge this PR.